paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/source/StaticFileStoreSplitEnumerator.java

```diff
@@ -37,6 +37,9 @@
 public class StaticFileStoreSplitEnumerator
         implements SplitEnumerator<FileStoreSourceSplit, PendingSplitsCheckpoint> {
 
+    /** Default batch splits size to avoid exceed `akka.framesize`. */
+    private static final int DEFAULT_SPLITS_SIZE = 10;
```

Comment:
Can we introduce an option for this? Can be an option in `FlinkConnectorOptions`?